src/ElementUtilities.js

```diff
@@ -21,7 +21,7 @@ const findClosest = (el, selector) => {
  * @return {boolean}        Whether table ancestor of 'el' is found
```

Comment:
This probably deviates from the original implementation in that it only considers the parent, not the passed in element. Consider replacing with the original and replacing usages with el.parentElement OR renaming to findClosestAncestor().